src/sentry/api/endpoints/group_details.py

```diff
@@ -236,16 +236,19 @@ def get(self, request: Request, group) -> Response:
                 )
                 data.update({"sentryAppIssues": sentry_app_issues})
 
-            if "hasAttachments" in expand:
+            if "latestEventHasAttachments" in expand:
```

Comment:
does this break any behavior on the issue details page?

Reply:
ex here https://github.com/getsentry/sentry/blob/79871b806059d17c3014ae0e9c235c544dac4da3/static/app/components/feedback/useFeedbackListQueryKey.tsx#L97

Reply:
got it 👍🏼